### PR TITLE
Gursaanj/dossier menu Functionality

### PR DESCRIPTION
**Pull request by Gursaanj**
Created Dossier Menu canvas with temp placeholder on the labels and image. Created simple UI control script for functionality

### CODE REVIEW

File: Assets/_GameJamCat/Scripts/UI/DossierMenu.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        private void OpenCloseMenu()
+        {
+            if (_isMenuOpen)
+            {
+                transform.localScale = Vector3.zero;
+            }
+            else
+            {
+                transform.localScale = Vector3.one;
```
Comment (m4rtint) [inline Assets/_GameJamCat/Scripts/UI/DossierMenu.cs]: transform.localScale = _isMenuOpen ? Vector3.zero : Vector3.one

Comment (Gursaanj) [reply Assets/_GameJamCat/Scripts/UI/DossierMenu.cs]: Good catch

**Comment by m4rtint** [comment]
Hey UI Text elements are kinda off, can you fix it?

**Review by m4rtint — CHANGES_REQUESTED** [review]
UI Changes needed

**Comment by Gursaanj** [comment]
> 
> Hey UI Text elements are kinda off, can you fix it?

Oh woah not sure how that came about, yeah will fix in like 30 min

**Review by m4rtint — APPROVED** [review]

**Pull request merged**